modules/csg/csg_shape.cpp

```diff
@@ -31,6 +31,8 @@
 #include "csg_shape.h"
 
 #include "core/math/geometry_2d.h"
+#include "manifold/common.h"
+#include "manifold/manifold.h"
```

Comment:
The explicit include for `manifold/common.h` could be removed too as it's already included by `manifold.h`.
Usually libraries like this are designed so that for the basic API there's a single header to include named like the library, and it comes with batteries included.
```suggestion
#include "core/math/geometry_2d.h"

#include <manifold/common.h>
#include <manifold/manifold.h>
```